architecture/core_concepts/routes.adoc

```diff
@@ -352,6 +354,118 @@ and deleted, the router adds and removes corresponding pool members.
 - When a user deletes the route and all endpoints associated with a particular
 pool, the router deletes that pool.
 
+[[architecture-f5-native-integration]]
+==== F5 Native Integration
+
+With new
+xref:../../install_config/router/f5_router.adoc#setting-up-f5-native-integration-with-openshift[native
+integration of F5 with {product-title}], you do not need to configure a ramp
+node for F5 to be able to reach the pods on the overlay network as created by
+OpenShift SDN.
+
+*Connection*
+
+F5 appliance can connect to the {product-title} cluster via an L3 connection. An
+L2 switch connectivity is not required between {product-title} nodes. On the
+appliance, you can use multiple interfaces to manage the integration:
+
+* Management interface - Reaches the web console of the F5 appliance.
+* External interface - Configures the virtual servers for inbound web traffic.
+* Internal interface - Programs the appliance and reaches out to the pods.
+
+image::F5-OpenShift-Connection-Diagram.png[F5 and OpenShift Connection Diagram]
+
+An F5 controller pod has `admin` access to the appliance. It is launched within
+the {product-title} cluster (scheduled on any node) that uses iControl REST APIs
+to program the virtual servers with policies, and configure the VxLAN device.
```

Comment:
I'm a little confused by the above. I think maybe the "it" is putting me off because I'm not too sure what it's referring to. But also, what are iControl REST APIs? Is that something OpenShift uses by default?